### ISSUE TITLE
have shared data for different client with same connection parameters

### ISSUE THREAD

**Issue by KashiMoonactive** [stipsan/ioredis-mock#464]
If creating 2 Redis clients with same connection parameters, I would like to have shared data between them
```javascript
const Redis = require('ioredis-mock');
const client1 = new Redis('localhost', 6378);
const client2 = new Redis('localhost', 6378);
await client1.hset('testing', 'test', '2');
const val = await client2.hget('testing', 'test');
console.log(val) // should output 2
```

As I see it, it can be implemented by mapping connection string to a DB object

**Comment by stipsan** [comment]
PRs welcome 😄

**Comment by insanehong** [comment]
Thanks :-)

**Comment by Parcley** [comment]
how about a feature like this:

```
const Redis = require('ioredis-mock');
const client1 = new Redis();
const client2 = client1.createConnectedClient();
await client1.hset('testing', 'test', '2');
const val = await client2.hget('testing', 'test');
console.log(val) // should output 2
```

**Comment by Parcley** [comment]
prototype at PR #656

**Comment by Parcley** [comment]
should be resolved?

**Comment by stipsan** [comment]
Indeed! Resolved in [v4.5.0](https://github.com/stipsan/ioredis-mock/releases/tag/v4.5.0) 🎉
